### PR TITLE
Add deletion of a staging to the best practices

### PR DESCRIPTION
**Pull request by krauselukas**
screenshot-2018-11-23 using the obs web ui best practice guide

### CODE REVIEW

File: xml/obs_best_practice_staging_workflow.xml [hunk 1]
```diff
@@ -80,6 +80,23 @@
       </listitem>
     </itemizedlist>
   </sect2>
+  <sect2>
+    <title>Delete a Staging</title>
+    <para>
+      By clicking on the delete button on the Staging index page, we are able
+      to delete a Staging. By selecting the associated staging projects
+      in the appearing window, we are able to delete them as well. If not selected,
```
Comment (dmarcoux) [inline xml/obs_best_practice_staging_workflow.xml]:
```suggestion
      in the modal window, we are able to delete them as well. If not selected,
```

Comment (krauselukas) [reply xml/obs_best_practice_staging_workflow.xml]: @dmarcoux i allways thought that "modal" is just a bootstrap term, but it actually makes sense :D i changed it

Comment (dmarcoux) [reply xml/obs_best_practice_staging_workflow.xml]: It should be `modal window`, not `appearing modal`.

Comment (krauselukas) [reply xml/obs_best_practice_staging_workflow.xml]: :sweat: done

**Pull request merged**